protonup-rs/src/download.rs

```diff
@@ -230,16 +234,13 @@ pub async fn download_to_selected_app(app: Option<apps::App>) {
     // versions_to_install = vec![];
 
     // Let the user choose which releases they want to use
-    let mut release_list = match helper_menus::multiple_select_menu(
+    let mut release_list = helper_menus::multiple_select_menu(
         "Select the versions you want to download :",
         release_list,
-    ) {
-        Ok(release_list) => release_list,
-        Err(e) => {
-            eprintln!("The tag list could not be processed.\nError: {}", e);
-            vec![]
-        }
-    };
+    ).unwrap_or_else(|e| {
+        eprintln!("The tag list could not be processed.\nError: {}", e);
+        vec![]
+    });
```

Comment:
I don't see a reason for this change.

Reply:
I'm not certain on this, but I think someone on the Rust discord said unwrap_or_else might be worse for performance because it will run the closure inside of else even if the unwrap is successful, which doesn't happen with a match. I'm not sure if any of that's true however.

Reply:
Not that performance would matter in this case anyways. But still, I might revert this back to a match someday.